### DISCUSSION TITLE
Github Pages deployment action stuck in queue

### DISCUSSION THREAD

**Discussion by vadimkantorov** [community/community#112469] [Actions]
### Select Topic Area

Bug

### Body

<img width="710" alt="image" src="https://github.com/community/community/assets/1041752/6f86136a-0a26-4385-a6d9-35b6c89abf89">

Canceling and re-triggering deployment does not help, the new action is also stuck in queue

**Comment by Bulat-Ziganshin** [comment]
you can search for "stuck" here, even in the FAQ. it often means incorrect "runs-on" spec, so it waits endlessly for an nonexistent runner
